### PR TITLE
Changes for CP-6150

### PR DESCRIPTION
**Pull request by Mona2012**

### CODE REVIEW

File: pages/04.HDM Doc 2.2/08.vcs/60.Undeploy/default.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Undelpoy Steps'
+title: 'Undeploy Steps'
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/08.vcs/60.Undeploy/default.md:2]: Is this change only required in pages & not in macros?

Comment (Mona2012) [reply pages/04.HDM Doc 2.2/08.vcs/60.Undeploy/default.md:2]: Yes this is only at one place in vcs pages.

Comment (kgcb) [reply pages/04.HDM Doc 2.2/08.vcs/60.Undeploy/default.md:2]: Ok, fine

**Review by kgcb — APPROVED** [review]
Looks good to me.

**Review by ayyajpio — APPROVED** [review]
Looks good to me.

**Pull request merged**